indico/modules/events/contributions/controllers.py

```diff
@@ -34,4 +56,55 @@ class RHManageContributions(RHManageContributionsBase):
     """Display contributions management page"""
 
     def _process(self):
-        return WPManageContributions.render_template('management/contributions.html', self.event)
+        contribs = Contribution.find_all(event_id=self.event.id, is_deleted=False)
+        sessions = Session.find_all(event_id=self.event.id, is_deleted=False)
+        tracks = self.event.getTrackList()
```

Comment:
@ThiefMaster, any idea here? We need to get `contribs`, `sessions` and `tracks` here so that the template can pass it to `render_contribution_list` in `_contribution_list.html`, exactly the same we do a few lines above.